### DISCUSSION TITLE
the difference between the number of columns for the input series (id of token) and the number of rows for the Attention weights

### DISCUSSION THREAD

**Discussion by YamamotoKensuke** [espnet/espnet#5189] [Q&A]
I am looking at the intermediate representation during generation in the Tacotron2 module, and I noticed that the number of columns for the input series (id of token) and the number of rows for the Attention weights are different. Am I correct in assuming that this is introducing a constant term? And in that case, is it correct to assume that the 0th corresponds to the constant term?

**Comment by kan-bayashi** [comment]
Maybe `<eos>` token, which is added at the end of sequences.
https://github.com/espnet/espnet/blob/13b986ce0595b4366c6e3381a981379600b958a4/espnet2/tts/tacotron2/tacotron2.py#L310-L314

**Reply by YamamotoKensuke** [reply]
Thank you!
